CHANGELOG.rst

```diff
@@ -1,6 +1,7 @@
 Changelog
 =========
 
+
```

Comment:
The person preparing a new release is not always the person who wrote the changes.

It is much more difficult to parse out a good summary from a diff after the fact, whereas if the original author provides their own summary, then the changelog is ready to go upon release